docs/pages/router/reference/static-rendering.mdx

```diff
@@ -283,6 +283,51 @@ export default function Page() {
 }
 ```
 
+## Fonts
+
+Expo Font has automatic static optimization for font loading in Expo Router. This means when you load a font with `expo-font`, Expo CLI will automatically extract the font resource and embed it in the page's HTML, enabling preloading, faster hydration, and reduced layout shift.
+
+The following block will load Inter into the namespace and statically optimize on web:
```

Comment:
```suggestion
Expo Font has automatic static optimization for font loading in Expo Router. When you load a font with `expo-font`, Expo CLI will automatically extract the font resource and embed it in the page's HTML, enabling preloading, faster hydration, and reduced layout shift.

The following snippet will load Inter into the namespace and statically optimize on web:
```

We can be concise and also let's use snippet instead of block since we use that word elsewhere in the docs.